contracts/src/v0.8/dev/VRFV2Wrapper.sol

```diff
@@ -0,0 +1,260 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+import "../vendor/SafeMathChainlink.sol";
+
+import "../ConfirmedOwner.sol";
+import "../interfaces/TypeAndVersionInterface.sol";
+import "../VRFConsumerBaseV2.sol";
+import "../interfaces/LinkTokenInterface.sol";
+import "../interfaces/AggregatorV3Interface.sol";
+import "../interfaces/VRFCoordinatorV2Interface.sol";
+import "../vendor/SafeMathChainlink.sol";
+import "./VRFV2WrapperInterface.sol";
+
+/**
+ * @notice A wrapper for VRFCoordinatorV2 that provides an interface better suited to one-off
+ * @notice requests for randomness.
+ */
+contract VRFV2Wrapper is ConfirmedOwner, TypeAndVersionInterface, VRFConsumerBaseV2, VRFV2WrapperInterface {
+  using SafeMathChainlink for uint256;
```

Comment:
I wonder if this is necessary for Solidity 0.8? I thought that it comes by default with 0.8 (https://soliditydeveloper.com/solidity-0.8)

Reply:
TIL! removed